R/data.R

```diff
@@ -0,0 +1,12 @@
+#' Demographic Dataset-updated
+#'
+#' An updated SDTM DM dataset with pediatric patients
+#'
+"dm_ped"
```

Comment:
Given that the package extension is {admiralpeds} I would use the _peds extension here.

Reply:
updated